spec/lib/coach/request_serializer_spec.rb

```diff
@@ -34,5 +35,39 @@
         expect(sanitized).to eq('value')
       end
     end
+
+    context "as an instance" do
+      before do
+        @mock_request = instance_double("Request", :request,
```

Comment:
What's the reason to use an instance variable here, rather than `let(:mock_request)`?